.github/workflows/release.yml

```diff
@@ -132,7 +130,7 @@ jobs:
           CIBW_BEFORE_BUILD_WINDOWS: pip install delvewheel
           CIBW_REPAIR_WHEEL_COMMAND_WINDOWS: delvewheel repair --add-path ${{ runner.temp }}\geos-${{ env.GEOS_VERSION }}\bin -w {dest_dir} {wheel}
           CIBW_TEST_REQUIRES: pytest
-          CIBW_TEST_COMMAND: pytest --pyargs pygeos.tests
+          CIBW_TEST_COMMAND: pytest --pyargs shapely.tests
```

Comment:
couldn't this simply be `pytest` without any other arguments? (similar for `.travis.yml`)

Reply:
Maybe? But we struggled quite a bit with that in the original PR https://github.com/pygeos/pygeos/pull/365, and this was eventually the result of different tries (this should ensure that the installed package is tested, and not the source. And we included the tests in the installed package)

So if we would want to revisit this, I prefer doing that in a separate PR :)